controllers/codeready_reconciler.go

```diff
@@ -26,14 +26,39 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+var CodeReadyLabels = map[string]string{
```

Comment:
If `CodeReadyLabels` is not getting used outside the package, please make it `codeReadyLabels`